src/OrleansRuntime/Streams/QueueBalancer/StreamQueueBalancerFactory.cs

```diff
@@ -21,7 +22,7 @@ internal class StreamQueueBalancerFactory
         /// <param name="siloMaturityPeriod">Maturity Period of a silo for queue rebalancing purposes</param>
         /// <returns>Constructed stream queue balancer</returns>
         public static IStreamQueueBalancer Create(
-            StreamQueueBalancerType balancerType,
+            string balancerType,
             string strProviderName,
             ISiloStatusOracle siloStatusOracle,
```

Comment:
ISiloStatusOracle, ClusterConfiguration, and IStreamProviderRuntime, can be obtained by DI in the factory's constructor.  We should probably remove them from the Create call.